website/homepage/src/docs/obj_doc_template.html

```diff
@@ -38,7 +38,7 @@
   {% else %}
   <div class="codeblock"><pre><code class="lang-python">{{ parent }}.<span>{{ obj["name"] }}(</span><!--
 -->{% for param in obj["parameters"] %}<!--
-  -->{% if "positional" in param["kind"] and "default" not in param %}<!--
+  -->{% if "positional" in param["kind"] and "default" not in param and param["name"] != "self" %}<!--
```

Comment:
Removes "self" from appearing in the list of parameters for every function in the docs page

Reply:
Ah yeah that looks weird. I think including all of the parameters is too much (especially for classes like `Interface`). How about we always include the first parameter (even if it has a default argument) + include all parameters that are required (aka don't have defaults)

Reply:
Will discuss that separately